### PR TITLE
reformatted text

### PR DESCRIPTION
**Pull request by vchernys**

### CODE REVIEW

File: cellprofiler/modules/imagemath.py [hunk 1]
```diff
@@ -171,28 +185,28 @@ def add_image(self, removable=True):
...
+You can perform math operations using two images or you can use a
+measurement for one of the operands. For instance, to divide the
+intensity of one image by another, choose *%(IM_IMAGE)s* for both and
+pick the respective images. To divide the intensity of an image by its
+median intensity, use **MeasureImageIntensity** prior to this module to
+calculate the median intensity, then select *%(IM_MEASUREMENT)s* and
+use the median intensity measurement as the denominator
+""" % globals()))
+
+        group.append("image_name", cellprofiler.setting.ImageNameSubscriber("", "", doc="""\
```
Comment (bethac07) [inline cellprofiler/modules/imagemath.py]: Please change to group.append("image_name", cellprofiler.setting.ImageNameSubscriber("Select the image", "", doc="""\

File: cellprofiler/modules/imagemath.py [hunk 2]
```diff
@@ -171,28 +185,28 @@ def add_image(self, removable=True):
...
-
-        group.append("factor", cellprofiler.setting.Float("", 1, doc="""
-            Enter the number that you would like to multiply the above image by. This multiplication
-            is applied before other operations."""))
+                "Measurement", lambda: cellprofiler.measurement.IMAGE, "", doc="""\
+This is a measurement made on the image. The value of the
+measurement is used for the operand for all of the pixels of the
+other operand's image."""))
+
+        group.append("factor", cellprofiler.setting.Float("", 1, doc="""\
```
Comment (bethac07) [inline cellprofiler/modules/imagemath.py]: Please change to group.append("factor", cellprofiler.setting.Float("Multiply the image by", 1, doc="""\

**Comment by bethac07** [comment]
(None of these changes are @vchernys fault by the way, just old errors in the documentation it'd be a great time to fix!)

**Comment by mcquin** [comment]
@vchernys To prevent the duplicate setting text, replace `help_settings` with:

```python
    def help_settings(self):
        return [
            self.operation,
            self.output_image_name,
            self.images[0].image_or_measurement,
            self.images[0].image_name,
            self.images[0].measurement,
            self.images[0].factor,
            self.exponent,
            self.after_factor,
            self.addend,
            self.truncate_low,
            self.truncate_high,
            self.ignore_mask
        ]
```

**Pull request merged**